### ISSUE TITLE
Can't put time value as child content of parseXXX tags

### ISSUE THREAD

**Issue by gmazza** [sargue/java-time-jsptags#4]
According to the documentation in the README for the "value" attribute for parseLocalDateTime (and the others), the "value" attribute is not required if the value is given as child content (Indeed, the code can process the body contentt: [1]).  However, when I try to do so:

                    <javatime:parseLocalDateTime>
                        <s:property value="firstComment.postTime" />
                    </javatime:parseLocalDateTime>

I get a complaint: "Caused by: org.apache.jasper.JasperException: According to the TLD or the tag file, attribute value is mandatory for tag parseLocalDateTime".  The issue is probably that the taglib defs[2] need to have the "required" property set from "true" to "false" for this to work.

[1] https://github.com/sargue/java-time-jsptags/blob/master/src/main/java/net/sargue/time/jsptags/ParseSupport.java#L165

[2] https://github.com/sargue/java-time-jsptags/blob/master/src/main/resources/META-INF/javatime.tld#L381

**Comment by gmazza** [comment]
Incidentally, a question I have on SO about how to upgrade from Struts2 tags to this one for formatting dates:  http://stackoverflow.com/questions/37777284/using-java8-time-in-jsps .  (If you can answer, great...)  This would make a nice FAQ item once it gets answered.

**Comment by sargue** [comment]
Hi. Thanks for catching this. It is indeed a bug on the documentation or the code. This project is a fork from Joda-time JSP tags and inherits most of its code and documentation. I've checked the parent project and it's broken there also.

Anyway something has to be done: updating documentation or reworking the code. I will try first to allow the tag without the value attribute and see if I can make that code work.

**Comment by gmazza** [comment]
Either way is good, but I might advise just updating the documentation to disallow child content, as that child-parsing code from JODA hasn't been activated so it might be buggy, and it doesn't offer users any more functionality that what they currently have.  Also, just having one way to do things helps reduce confusion during Google searches as everyone gets steered on to the same path.  

Anyway, I figured out how to convert from Struts tags (s:date and s:text) to your javatime:format, your taglib is now being used within my "TightBlog" open source project.  I've updated the SO link above to fully explain to others.
